ansible-ipi-install/roles/installer/tasks/main.yml

```diff
@@ -2,6 +2,10 @@
 # tasks file for installer
 - include_tasks: 10_get_oc.yml 
 - include_tasks: 20_extract_installer.yml
+- include_tasks: 23_rhcos_image_paths.yml
+  when: ((release_version[0]|int == 4) and (release_version[2]|int <= 3)) or cache_enabled == true
+- include_tasks: 24_rhcos_image_cache.yml
+  when: cache_enabled == true
```

Comment:
just confirming behavior: we expect 24_* to always run if cached_enabled is true? Btw I'd change these to uppercase True

Reply:
That's correct.